doc/conf.py

```diff
@@ -22,8 +22,8 @@
 project = u'metric-learn'
 copyright = u'2015-2018, CJ Carey and Yuan Tang'
 author = u'CJ Carey and Yuan Tang'
```

Comment:
I think this line needs some updating as well!

Reply:
That's right, done, for the order I thought [original developers | my supervisors | me]

Reply:
Please put yourself before Nathalie and I!

Reply:
Allright, done ;)